src/components/security_manager/src/security_manager_impl.cc

```diff
@@ -241,6 +354,38 @@ void SecurityManagerImpl::RemoveListener(
   listeners_.remove(listener);
 }
 
+bool SecurityManagerImpl::OnCertificateUpdated(const std::string& data) {
+  LOG4CXX_AUTO_TRACE(logger_);
+  LOG4CXX_DEBUG(logger_, "Certificate updated");
```

Comment:
@AByzhynar debug is redundant

Reply:
@LuxoftAKutsan Changed in eb15dd852a0ea418bfa444b74c8f15d792e254ad